packages/react/src/lib/positioner/Popper.tsx

```diff
@@ -120,7 +120,7 @@ const Popper: React.FunctionComponent<PopperProps> = props => {
       popperRef.current = createPopper(targetRef.current, contentRef.current, options)
     },
     // TODO review dependencies for popperHasScrollableParent
-    [computedModifiers, enabled, userModifiers, positionFixed, proposedPlacement],
+    [computedModifiers, enabled, userModifiers, positionFixed, proposedPlacement, targetRef],
```

Comment:
@Bugaa92 @layershifter please take a look on this change. I needed this in order for the Tooltip with custom target to work. Should we add here targetRef.current instead (or together with this)?

Reply:
It's correct, @Bugaa92 will attach an updated example's code

Reply:
I investigated this with @layershifter and this change is correct but the examples need to be updated, pls take a look at https://github.com/stardust-ui/react/pull/1455/files#r292519287